### DISCUSSION TITLE
new tool "button as output as keyboard" for third parties such as office word, notepad, etc on the live transcribe web

### DISCUSSION THREAD

**Discussion by dpomeranec** [speechmatics/community#11] [Ideas & suggestions]
is there any possible to add on the live transcribe web the new function "button as output as keyboard" so I can transfer the live transcribe to the third party such as office word, notepad, etc ?

**Comment by TudorCRL** [comment]
Hey @dpomeranec, thanks for reaching out to us!

Just for clarification, I wanted to check exactly what you were asking for - do you want to know if it's possible to copy the output of our realtime transcription service on our portal website and paste it elsewhere, or are you looking for a way to pipe that output in realtime to some other service such as word or notepad?

In the case of the former, you should be able to copy the contents using the copy button once you're finished transcribing, as in the screenshot:
<img width="837" alt="Screenshot 2023-09-04 at 16 56 58" src="https://github.com/speechmatics/community/assets/104087420/4113b5fd-fe85-4b2d-8092-6d22cfeb2c62">

In the case of the latter, there's not an easy way to do this from the portal UI unfortunately. However, if what you're looking for is a way to transcribe yourself talking and to pipe it directly out to a file, then this could be achieved using our CLI tool or one of our SDKs. You can find examples of how this is done [here](https://docs.speechmatics.com/introduction/rt-guide#real-time-python-example).

You can also find a good tutorial on using FFMPED to pipe audio into our CLI tool [here](https://docs.speechmatics.com/tutorials/using-ffmpeg). The upshot is that, for example on mac, you can pipe audio from the CLI with this single command:

```
ffmpeg -loglevel quiet -f avfoundation -i ":default" -f wav - | \
    speechmatics transcribe --max-delay 2 --profile prod - > out.txt
```

Sorry I've probably given a bit too much info there, but hopefully it helps! If that doesn't answer you're question, let me know and I'll try to refine my answer a bit

**Comment by dpomeranec** [comment]
Hi thank you for your reply
Again I am going to explain more clear to you what I wish that the
sppechmatic webpage would have a special button to allow us to pipe that
output in realtime to some other service such as word or notepad
It is similar to the stenographer or dragon naturally softwares
Regards
Diego

**Reply by TudorCRL** [reply]
Okay, I see - thanks for the clarification. I'd not seen Dragon NaturallySpeaking before, that's an interesting use case. We don't currently support this, but it's really great to get your feedback on what kind of use case you'd like to see. It'd be interesting to know a bit more about the context in which you'd like to use it - are you looking for a professional service or something that you want for personal use to help you do some simple text document dictation?

**Comment by dpomeranec** [comment]
I am using iCap app where I can type o use voice speech, that connects to
closed captioning encoder. That is why I need to use the Speechmatics to
implement into iCap app but the speechmatics doesn’t have any function to
send the live transcribe to the third party apps
Diego

**Reply by TudorCRL** [reply]
Okay, I see. In general, Speechmatics isn't a B2C business so we don't tend to provide these kinds of end user solutions for people. We're more focussed on providing the underlying API and the services around that, like SaaS, On Prem, developer platform and so forth. For your use case it sounds like you might be better off looking at some of the more consumer-focussed offerings on the market.
